docker-compose.yml

```diff
@@ -18,13 +18,30 @@ services:
     ports:
       - 27017:27017
 
+  mysql:
+    image: mysql:8.0.23
```

Comment:
version used in aws

Reply:
does this change at all over time? or does it not matter

Reply:
it does get updates - we'll update it manually as the aws one updates